sensu/plugins/check-vyatta-tunnels.py

```diff
@@ -59,11 +59,11 @@ def main():
         switch_on_criticality()
 
     for index, device in enumerate(device_list):
-        if " vti" in str(device):
+        if "vti" in str(device):
             status += "Device: %s " % device.split("STRING: ")[1].rstrip()
             status += "%s\n" % device_status[index].split("INTEGER: ")[1]
 
-    if "down" in status:
+    if "down" in status or status == DOWN:
```

Comment:
Can this be `if DOWN in status`?  Since we have found that the integer return is consistent I dont see any point in checking for the string as well.